templates/serverpod_templates/projectname_server/lib/src/greeting.spy.yaml

```
@@ -6,18 +6,19 @@
 # Please consult the documentation for more information on what you can add to
 # your yaml-files.
 
-
-# Name of the class to generate.
-class: Example
+### A greeting message which can be sent to the client
+class: Greeting
 
 # Add the table key, if this class represents a row in the database.
-#table: example
+#table: greeting
 
-# The fields (and columns if connected to the database) of the class. Supported types are 
-# `bool`, `int`, `double`, `String`, `UuidValue`, `Duration`, `DateTime`, `ByteData`, 
-# and other serializable classes, exceptions and enums. 
-# You can also add lists of objects and types have support for null safety. 
-# Eg. `List<int>?` or `List<MyOtherClass?>`.
+# The fields (and columns if connected to the database) of the class.
+# For a list of supported types, please see the documentation.
+# https://docs.serverpod.dev/concepts/working-with-endpoints
 fields:
-  name: String
-  data: int
\ No newline at end of file
+  ### The greeting message
```

```suggestion
  ### The greeting message.
```

Let's be consistent with punctuations throughout our code and examples.